client/blocks/eligibility-warnings/index.tsx

```diff
@@ -46,6 +46,7 @@ interface ExternalProps {
 	siteId?: number | null;
 	isEligible?: boolean;
 	backUrl?: string;
+	onDismiss?: () => void;
```

Comment:
This component isn't always displayed in a modal, so this prop is optional in cases where something needs to happen when the user doesn't proceed and the warnings are dismissed. In the plugins case it is closing the modal.